frontend/mr-admin-flate/src/pages/gjennomforing/utbetaling/OpprettUtbetalingForm.tsx

```diff
@@ -74,17 +74,13 @@ const Schema = z
   )
   .refine(
     (data) => {
-      const KID_REGEX = /^\d{2,25}$/;
-      return !data.kidNummer || KID_REGEX.test(data.kidNummer);
+      if (!data.kidNummer) {
+        return true;
+      } else {
+        return data.kidNummer && KID_REGEX.test(data.kidNummer);
+      }
     },
     { message: "KID-nummer må være mellom 2 og 25 siffer", path: ["kidNummer"] },
```

Comment:
Kanskje egentlig en fovirrende feilmelding? En kid på to siffer er vel egentlig aldri gyldig? Kanskje bare fjerne all frontend-validering utenom required?

Reply:
Godt poeng. Jeg fjernet sjekken fra frontend helt, siden den er i backend allerede